lib/Doctrine/DBAL/Platforms/SQLAnywherePlatform.php

```diff
@@ -551,7 +551,7 @@ public function getDropIndexSQL($index, $table = null)
 
         if (! is_string($index)) {
             throw new InvalidArgumentException(
-                'SQLAnywherePlatform::getDropIndexSQL() expects $index parameter to be string or ' . Index::class . '.'
+                sprintf('SQLAnywherePlatform::getDropIndexSQL() expects $index parameter to be "string" or an instance of %s.', Index::class)
```

Comment:
Should it be "…a string or an instance…"?

Reply:
So no quotes around string?

Reply:
No quotes. We're not quoting a value "string", we're referring to a type.